about.css

```diff
@@ -0,0 +1,267 @@
+  z-index: -100;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+#about-heading {
+  font-size: 35px;
+  background-image: linear-gradient(to bottom right, rgba(236, 82, 66, 0.8) 40%, rgba(236, 82, 66, 0.5) 60%), url('./Assets/background.jpg');
+  background-repeat: repeat;
+  background-size: contain;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+#about-description {
+  padding: 12px 16px;
+  border: 4px solid #eee;
+  background-color: #f5f5f5;
+  box-shadow: rgb(0 0 0 / 20%) 0 0 6px;
+}
+
+.about-contact-info {
+  padding: 20px;
+  font-size: 18px;
+}
+
+.about-initiative {
+  text-align: center;
+  color: #565656;
+  padding: 25px 15px;
+}
+
+#event-container {
+  gap: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.events {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  justify-items: center;
+  background-color: #f2f2f2;
+  padding: 20px;
+  gap: 30px;
+  margin: auto;
+}
+
+.event-card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: center;
+  color: #fff;
+  padding: 20px;
+  min-width: 340px;
+  min-height: 300px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+#event1 {
+  background-image: linear-gradient(to bottom, rgba(236, 82, 66, 0.8) 20%, rgba(236, 82, 66, 0.8) 70%), url('./Assets/bootcamp.avif');
+}
+
+#event2 {
+  background-image: linear-gradient(to bottom, rgba(236, 82, 66, 0.8) 20%, rgba(236, 82, 66, 0.8) 70%), url('./Assets/bootcamp2.avif');
+}
+
+.event-text {
+  color: white;
+  text-align: center;
+  margin: auto;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  background-color: black;
+  color: #f2f2f2;
+  padding: 20px;
+}
+
+#footer-text {
+  text-align: center;
+}
+
+/* desktop version */
+
+@media screen and (min-width: 750px) {
+  /* nav session */
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 100px 20px 20px;
+  }
+
+  .close-menu * {
+    padding-right: 20px;
+  }
+
+  #logo {
+    font-size: 20px;
+    font-weight: 900;
+  }
+
+  #nav-links {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 20px;
+    font-size: 18px;
+    font-weight: 600;
+  }
+
+  #nav-links *:hover {
+    cursor: pointer;
+    color: #ec5242;
+  }
+
+  #last {
+    padding: 0 20px;
+    border: 4px solid #ec5242;
+    color: #ec5242;
+  }
+
+  /* header session */
+  #heading {
+    display: flex;
+    flex-direction: column;
+    padding-top: 200px;
+    justify-content: center;
+    text-align: center;
+  }
+
+  #about-heading {
+    font-size: 70px;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 70%;
+  }
+
+  #about-description {
+    max-width: 50%;
+    margin-left: auto;
+    margin-right: auto;
+    font-size: 24px;
+    padding: 20px;
+  }
```

Comment:
- Please, kindly use the correct color of #515151 and a font size of 16px  to match the design as much as you can.